views/application.py

```diff
@@ -1,17 +1,21 @@
 from rest_framework import response
-from rest_framework import viewsets, permissions
+
```

Comment:
I meant that you need to group them. And there should be no line breaks amongst the group members.

```python
from rest_framework import response
from rest_framework import viewsets, permissions

from open_auth.models import Application
from open_auth.serializers.application import (
    ApplicationDetailSerializer,
    ApplicationListSerializer,
)

from developer.utils.membership_notifications import (
    send_membership_notification
)
```